doc/source/classifier_tutorial.rst

```diff
@@ -0,0 +1,116 @@
+.. Copyright © 2023 Ernst Strüngmann Institute (ESI) for Neuroscience
+.. in Cooperation with Max Planck Society
+
+.. SPDX-License-Identifier: CC-BY-NC-SA-1.0
+
+Parallel Bootstrap Tutorial
+---------------------------
+
+The following Python code demonstrates how to use ACME to perform a parallel bootstrap of the classification accuracy of three different scikit-learn classifiers.
+
+We start by loading the wine dataset from scikit-learn and splitting it into training and testing sets using the `train_test_split` function from `sklearn.model_selection`.
+
+.. code-block:: python
+
+    from sklearn import datasets
+    from sklearn.model_selection import train_test_split
+
+    data = datasets.load_wine()
+    X_train, X_test, y_train, y_test = train_test_split(data.data, data.target, test_size=0.5, random_state=42)
+
+
+Training and evaluating the classifiers
+---------------------------------------
+
+Next, we define three scikit-learn classifiers: K nearest neighbors, a NeuralNet, and support vector machine. 
+We train each of them on the training set and evaluate their accuracy on the test set using the `score` method.
+
+.. code-block:: python
+
+    from sklearn.neighbors import KNeighborsClassifier
+    from sklearn.neural_network import MLPClassifier
+    from sklearn.svm import SVC
+
+    neighbors = KNeighborsClassifier(n_neighbors=3)
+    NeuralNet = MLPClassifier(random_state=1, max_iter=300)
+    SVM       = SVC(gamma='auto')
+
+    neighbors.fit(X_train, y_train)
+    NeuralNet.fit(X_train, y_train)
+    SVM.fit(X_train, y_train)
+
+    print(f"K nearest neighbors accuracy: {neighbors.score(X_test, y_test):.3f}")
+    print(f"Naive Bayes  accuracy: {NeuralNet.score(X_test, y_test):.3f}")
+    print(f"Support Vector Machine accuracy: {SVM.score(X_test, y_test):.3f}")
+
+Because we evaluated the accuracy on the test data, we only get one accuracy measure per classifier.
+However, we would like to have a distribution of accuracies to later compare the confidence intervals.
+To achieve this, we can use `bootstrapping <https://en.wikipedia.org/wiki/Bootstrapping_(statistics)/>_`.
+
+Bootstrapping and confidence intervals
+-----------------------------------------
+
+We define a function `bootstrap_model_accuracy` that resamples the test set *with* replacement and calculates the accuracy of each classifier on the resampled data.
+We will use ACME to parallelize the bootstrapping process for efficiency.
+
+.. code-block:: python
+
+    from sklearn.utils import resample
```

Comment:
This is a really great example! Which version of ACME have you used to run it? The most recent ACME release also supports the keyword `result_shape` to store stuff directly in a (pre-allocated) NumPy array :)